changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 10.0.2
```

Comment:
This should probably be `10.1.0` if we're using `Added` in the changelog.

Reply:
True. Fixed.